gravity-forms/gw-require-unique-values.php

```diff
@@ -85,6 +94,34 @@ public function validate( $result, $value, $form, $field ) {
 		return $result;
 	}
 
+	public function validate_list( $result, $value ) {
+		$rows      = count( $value );
+		$is_unique = true;
+
+		// Multi-Column List
+		if ( is_array( $value[0] ) ) {
```

Comment:
Will this throw errors if `$value` is empty? I think you could use `rgar( $value, 0 )` if it is.